package.json

```diff
@@ -25,6 +25,7 @@
     "grunt-jsonlint": "^1.0.4",
     "grunt-jsxhint": "^0.6.0",
     "grunt-lintspaces": "^0.7.0",
+    "node-lintspaces": "shaoshing/node-lintspaces",
```

Comment:
yes the new package is in use. `grunt-lintspaces` relies on `node-lintspaces` and the later one does not check lines with spaces or tabs. I made a tiny fix to its regexp to address these lines.